src/GovUk.Education.ExploreEducationStatistics.Data.Services/SubjectMetaService.cs

```diff
@@ -122,5 +173,59 @@ private dynamic GetGeoJsonForObservationalUnit(long boundaryLevelId, IObservatio
             var geoJson = _geoJsonService.Find(boundaryLevelId, observationalUnit.Code);
             return geoJson != null ? JsonConvert.DeserializeObject(geoJson.Value) : null;
         }
+        
+        private IEnumerable<FootnoteViewModel> GetFootnotes(IQueryable<Observation> observations,
+            SubjectMetaQueryContext queryContext)
+        {
+            return _footnoteService.GetFootnotes(queryContext.SubjectId, observations, queryContext.Indicators)
+                .Select(footnote => new FootnoteViewModel
+                {
+                    Id = footnote.Id,
+                    Label = footnote.Content
+                });
+        }
+        
+        
+        private sealed class FilterEqualityComparer : IEqualityComparer<Filter>
```

Comment:
Consider extending AbstractTableBuilderSubjectMetaService instead for the next three private methods.